site/about.html

```diff
@@ -1,28 +1,42 @@
+<!doctype html>
+
 <html>
 	<head>
 		<title>About me</title>
-		<link rel="stylesheet" href="style.css" >
+
+		<link rel="stylesheet" href="style.css">
+		<link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
+		<link href='https://fonts.googleapis.com/css?family=Lato:400,900' rel='stylesheet' type='text/css'>
 	</head>
 
 	<body>
-		<nav>
-			<a href="about.html">About Me</a>
-			<a href="portfolio.html">Portfolio</a>
-		</nav>
+		<ul class="top_nav">
+			<li class="top_nav_item"><a href="about.html" class="active">About me</a></li>
+			<li class="top_nav_item"><a href="portfolio.html" class="li a">Portfolio</a></li>
+			<li class="top_nav_item"><a href="another_page.html" class="li a">Another Page</a></li>
+		</ul>
 
-		<h1>Joni Halabi</h1>
-		<h2>Developer. Cookie Baker. Whovian.</h2>
+		<div class="name_area">
+			<h1>I'M JOE JOHNSON.</h1>
+		</div>
 
-		<p class="greenparagraph">Hi!  I am Joni, a web developer in <span>Washington DC</span>, via New Jersey and Boston.  I have 15 years of experience in web development, specializing in retail and small business websites.</p>
+		<div class="description">
+			<div class="joe_pic">
+				<img src="images/joe.png" alt="Joe Johnson photo" width="250"> 
+			</div>
 
-		<p>This is another paragraph.</p>
+			<h3>I enjoy life as a developer.</h3>
 
-		<img src="images/me.jpg" />
+			<p>I have ten years of experience in the graphic design world, specializing in the creation of responsive websites. I've worked with clients including Nike, Target, Best Buy, Georgetown University and The Nature Conservancy. Originally from Chicago, I now live in NYC with my dog, Arthur. I enjoy cooking and collecting vinyl.</p>
+		</div>
+
+		<nav class="social_media">
```

Comment:
Great use of the ```<nav>``` tag in this block.  (It is OK to have more than 1 navigation block on a page, as long as both blocks contain navigational links.)